frontend/src/views/MyProgress/index.vue

```diff
@@ -1,88 +1,117 @@
+            <span v-if="campaignEndDate">
+              jusqu’au
+              {{ campaignEndDate.toLocaleString("fr-FR", { month: "long", day: "numeric", year: "numeric" }) }}.
             </span>
             <span v-else>
-              En cas d'erreur, veuillez
-              <router-link :to="{ name: 'ContactPage' }" class="grey--text text--darken-4">nous contacter</router-link>
-              .
+              jusqu’à la fin de la campagne.
             </span>
           </p>
-          <v-card-actions class="px-0 pt-0 pb-0 align-start d-block d-md-flex">
-            <div>
-              <DownloadLink
-                :href="`/api/v1/teledeclaration/${diagnostic.teledeclaration.id}/document.pdf`"
-                label="Télécharger le justificatif"
-                sizeStr="60 Ko"
-                target="_blank"
-                class="mb-0 mr-4"
-              />
+          <TeledeclarationCancelDialog
+            v-model="cancelDialog"
+            v-if="inTeledeclarationCampaign"
+            @cancel="cancelTeledeclaration"
+            :diagnostic="diagnostic"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn outlined small color="primary" class="fr-btn--tertiary px-2" v-on="on" v-bind="attrs">
+                Annuler ma télédéclaration
+              </v-btn>
+            </template>
+          </TeledeclarationCancelDialog>
+        </div>
+        <!-- TODO: satellites who are being declared for -->
+        <div v-else-if="inTeledeclarationCampaign">
+          <div v-if="readyToTeledeclare">
+            <DataInfoBadge class="my-2" :readyToTeledeclare="true" />
+            <div v-if="hasFinishedMeasureTunnel">
+              <p>Votre bilan est complèt !</p>
+              <v-btn color="primary" @click="showTeledeclarationPreview = true">Télédéclarer</v-btn>
             </div>
-            <div class="mt-4 mt-md-0">
-              <TeledeclarationCancelDialog
-                v-model="cancelDialog"
-                v-if="inTeledeclarationCampaign"
-                @cancel="cancelTeledeclaration"
-                :diagnostic="diagnostic"
-              >
-                <template v-slot:activator="{ on, attrs }">
-                  <a class="ml-0 ml-md-4 text-decoration-underline" v-on="on" v-bind="attrs">
-                    Annuler ma télédéclaration
-                    <v-icon color="primary" size="1rem" class="ml-0 mb-1 close-icon">
-                      $close-line
-                    </v-icon>
-                  </a>
-                </template>
-              </TeledeclarationCancelDialog>
+            <div v-else>
+              <p>Merci d'avoir pris le temps de saisir les données d'approvisonnement et de votre établissement.</p>
+              <p>Pour aller plus loin, vous pouvez complèter le bilan. Sinon, vous pouvez le télédéclarer.</p>
+              <v-btn outlined color="primary" @click="showTeledeclarationPreview = true">
```

Comment:
I didn't want to make it primary, because there is another primary button (which you have to scroll to see) on the not-started (and eventually, imo, not completed) tabs

Reply:
Let's leave it like that and think about it on a second PR - which could also add the "Pick up where you left off" button as well :+1: